File: Editorial/0119_Pascal_s_Triangle_II/TypeScript/1_Brute_Force_Recursion.ts
function getNum(row: number, col: number): number {
    if (row === 0 || col === 0 || row === col) {
        return 1;
    }
    return getNum(row - 1, col - 1) + getNum(row - 1, col);
}
function getRow(rowIndex: number): number[] {
    let ans: number[] = [];
    for (let i = 0; i <= rowIndex; i++) {
        ans.push(getNum(rowIndex, i));
    }
    return ans;
}

File: Editorial/119_Pascal_s_Triangle_II/CSharp/1_Brute_Force_Recursion.cs
public class Solution {
    private int getNum(int row, int col) {
        if (row == 0 || col == 0 || row == col) {
            return 1;
        }

        return getNum(row - 1, col - 1) + getNum(row - 1, col);
    }

    public IList<int> GetRow(int rowIndex) {
        IList<int> ans = new List<int>();
        for (int i = 0; i <= rowIndex; i++) {
            ans.Add(getNum(rowIndex, i));
        }

        return ans;
    }
}
